frontend/src/metabase/components/Popover.jsx

```diff
@@ -20,12 +21,13 @@ export default class Popover extends Component {
   constructor(props, context) {
     super(props, context);
 
+    this.handleDismissal = this.handleDismissal.bind(this);
     this.state = {
-      width: null,
-      height: null,
```

Comment:
These state values were not used in the component so I removed them and the associated `setInternal` call that tried to update them every 100ms.

Reply:
Was that 100ms interval the cause of incremental re-rendering of tethered elements?

![ ](https://p58.f1.n0.cdn.getcloudapp.com/items/8Lun8PeZ/c6781fb5-ed66-4e96-9f5c-30cd94186d80.gif?source=viewer&v=e3d0c8efa2af058d54c13f203fde4834)

Reply:
Removing the `setInterval` fixes some of the re-renders but not all of them.